app/Models/User.php

```diff
@@ -415,6 +417,16 @@ public function registerMediaCollections(): void
     {
         $this->addMediaCollection('avatar')
             ->singleFile();
+
+        $this->addMediaCollection('signature')
+            ->singleFile()
```

Comment:
I don't think this should be `singleFile`. Someone could have more than one image in their signature if they wanted I believe.